src/main/java/com/example/artvswar/controller/ControllerExceptionHandler.java

```diff
@@ -39,7 +39,8 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
         AllErrorFieldResponse response = new AllErrorFieldResponse();
         response.setTimestamp(new Date());
         response.setStatus(status.value());
-        response.setError(status.getReasonPhrase());
+        // todo test this "status.toString()"
```

Comment:
It is not a good practice to leave comments, If you want to have todo, then write it like this:
`todo: test this "status.toString()"` without commenting (//)